### PR TITLE
Test/graphql

### PR DESCRIPTION
**Pull request by aliroemhildt**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
- installed graphQL
- installed apollo client library
- created sampleGraphQL file that has a sample graphQL request
  
What (if any) features are you implementing?
- no new features, just installing

What (if anything) did you refactor?
- n/a

Were there any issues that arose?
- n/a

Is there anything that you need from your teammate?
- n/a

Any other comments, questions, or concerns?
- circleCI is working now, looks like it just took a little bit for the permissions to update

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+});
+
 ReactDOM.render(
-  <React.StrictMode>
+  <ApolloProvider client={ client }>
     <App />
-  </React.StrictMode>,
+  </ApolloProvider>,
   document.getElementById('root')
 );
```
Comment (mr-ryan12) [inline src/index.js:18]: Thank you for sharing the initial setup with us!

File: src/sampleGraphQL.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+//   }
+
+//   return (
+//     <div className="App">
+//       <p>{launchInfo}</p>
+//     </div>
+//   );
+// }
+
+// export default App;
```
Comment (mr-ryan12) [inline src/sampleGraphQL.js:44]: Using this as a reference to GraphQL/Apollo Client syntax

**Comment by Echory** [comment]
Looks good

**Review by Echory — APPROVED** [review]

**Review by oacortez — APPROVED** [review]
Thank you for creating this testing branch. I see we have a basic example of Apollo thank you for building that!

**Review by mr-ryan12 — APPROVED** [review]
Looks good thank you!

**Pull request merged**